app/code/community/Drip/Connect/Helper/Data.php

```diff
@@ -132,6 +132,22 @@ public function isCustomerExists($email, $websiteId = null)
         return (bool) $customer->getId();
     }
 
+    /**
+     * find customer by email address
+     *
+     * @return Mage_Customer_Model_Customer $customer
```

Comment:
Could we add the two params to the docblock?

Also, you generally don't add a variable name to a `@return` definition.
```suggestion
     * @return Mage_Customer_Model_Customer
```